### PR TITLE
Back to dev again

### PR DESCRIPTION
**Pull request by itrujnara**
## PR checklist

Merge of patch content from master and version bump to 1.1dev

- [x] This comment contains a description of changes (with reason).
- [x] Make sure your code lints (`nf-core lint`).
- [x] `CHANGELOG.md` is updated.

### CODE REVIEW

**Review by mashehu — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,15 +3,14 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [v1.1dev]
+## [v1.1]
```
Comment (mashehu) [inline CHANGELOG.md]:
```suggestion
## [v1.1.0]
```
Please use full semver

**Review by mashehu — CHANGES_REQUESTED** [review]

**Review by JoseEspinosa — APPROVED** [review]
LGTM!

**Review by mashehu — COMMENTED** [review]
I am a bit confused is to a version bump from a new release back to dev or from a prep for a new release?

**Comment by itrujnara** [comment]
> I am a bit confused is to a version bump from a new release back to dev or from a prep for a new release?

We had a patch because of Megatest issues, then when we merged the changes back into dev the version number 1.0.1 got merged as well and we need it to be 1.1dev here for the time being

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,15 +3,14 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [v1.1dev]
+## [v1.1.0]
```
Comment (mashehu) [inline CHANGELOG.md]:
```suggestion
## [v1.1.0dev]
```
like this then?

Comment (mashehu) [reply CHANGELOG.md]: (better to actually use `nf-core bump-version 1.1.0dev` to also edit the mutliqc config correctly)

**Review by mashehu — APPROVED** [review]

**Comment by itrujnara** [comment]
Template merge hell has started here, can someone more experienced help?

**Comment by itrujnara** [comment]
Force-merging with failing tests – the errors are due to unmerged 3.0 template

**Pull request merged**